cli.md

```diff
@@ -34,9 +34,12 @@ File group(s) used as output.
 
 ## Optional parameters
 
-### `-g, --group-id GROUPID`
+### `-g, --group-id ID`
 
-The `mets:file@GROUPID` to process. If no groupID is defined all files within the `inputGrp` will be processed. Repeatable and multiple IDs can be provided separated by comma.
+The `ID` of the `mets:div[@TYPE='page']` that contains the pointers to files
+representing a page. Effectively, only those files in the [input
+group](#-i---input-file-grp-grp) that are referenced in these
```

Comment:
Suggestion: Effectively, only those files in the [input file group] (#....)

Reply:
:+1: